stage-settings.md

```diff
@@ -19,24 +19,64 @@ Once a stage is added, administrators can edit the settings of that stage by sel
-If you choose both screening and annotation, you will be asked to choose how to combine these tasks under Study Selection Settings. 
+
+## Review Mode
+
+Under "Review Mode", project administrators can configure various settings for stages with screening and annotation, such as: include and decide which capabilities to add to your stage, e.g. screening, annotation and data extraction, by selecting these modes using the check boxes.  
+
+### Screening
+Screening Settings can be edited by clicking the link ‘Click for editing screening settings’. If you choose both screening and annotation, you will be asked to choose how to combine these tasks under "Study Selection Settings". 
 
 Specifically, the options are: 
 - All studies require screening, but annotation is optional 
 - All studies require annotation, but screening is optional 
 - All studies require both screening and annotation
 
-Read more about these options in [Stages.](../stages\.html)
-
+Read more about these options in [Stages.](../stages.html)
 
 ![Stage Options](/figs/Fig_stage_options.png)
 
 
-Note that it is not possible to delete a Stage once it has been added to project. It is only possible to edit the settings of the stage including disabling the stage so that it is not available to Reviewers on the project.
\ No newline at end of file
+
+### Annotation
+If annotation is selected, multiple options are presented. 
+
+__Number of required annotations per study__ - Project administrators can adjust the “number of required annotations per study” as it fits the respective project. This represents the number of independent reviewers who can annotate each study. Adjust the number in the input box by using the arrows or simply type a number. This input box only accepts whole numbers, or integers. We recommend two independent reviewers to annotate each study in your review as this is typically required from publication in scientific journals and this is the default option. For student-related projects, the option to decrease the number of reviewers is also possible. 
+
+![Advanced Annotation Settings](/figs/Stage-Advanced-settings-annotation-settings.png)
+
+
+__Maximum number of incomplete sessions per annotator__ - Project administrators can define how many annotations sessions with the status of “in progress” a reviewer on their project can have. When a reviewer reaches the limit, they will not be able to take new studies for annotation without completing previous incomplete or “saved” studies. Therefore, the lower the number here, the number of studies with the status of “in progress”, the fewer studies a reviewer can have incomplete and saved.  Adjust the number in the input box by using the arrows or simply type a number. This input box only accepts whole numbers, or integers. Currently the default is set to 100 papers, to facilitate reviewers to complete studies in a more timely manner, reduce the number.
```

Comment:
annotation sessions with the status of "in progress" each reviewer on their project can have.
reaches this limit
they will not be able to begin annotation of further studies without completing annotations on these "in progress" or saved studies.
Therefore, this number is the maximum number of "in progress" or studies with incomplete annotations a reviewer can have saved.
Currently, the default is set to 100 papers. To, reduce the number of studies a reviewer can have in progress at one time, reduce this number.